src/pybert/gui/view.py

```diff
@@ -812,25 +866,60 @@ def do_load_data(self, info):
             label="Jitter",
             id="jitter",
         ),
-        Group(  # Help
+        Group(  # Info
             Group(
-                Item("ident", style="readonly", show_label=False),
                 Item("perf_info", style="readonly", show_label=False),
-                label="About",
+                label="Performance",
             ),
-            Group(Item("instructions", style="readonly", show_label=False), label="Guide"),
+            Group(Item("instructions", style="readonly", show_label=False), label="User's Guide"),
             Group(Item("console_log", style="readonly", show_label=False), label="Console", id="console"),
             layout="tabbed",
-            label="Help",
-            id="help",
+            label="Info",
+            id="info",
         ),
         layout="tabbed",
         springy=True,
         id="tabs",
     ),
     resizable=True,
     handler=MyHandler(),
-    buttons=[run_sim, save_cfg, load_cfg, save_data, load_data],
```

Comment:
It looks like we never actually had a "Stop" button in the interface.
Although, I see it defined above.
Was that something that you had started to add, before you switched to a menu based approach?

Reply:
I don't think a stop button has been exposed as long as I can remember but the code was always there.  I've never used or modified it.

Reply:
Back in 2018 was added but not to the buttons!  https://github.com/capn-freako/PyBERT/commit/ecb226cec08a87d34eab688adef00209670a8ca7

Reply:
D'oh!
Well, I guess if we've never missed it then we don't _really_ need it.
I'll add it to the _Issues_ list, as a non-gating feature request for later development.